worlds/witness/hints.py

```diff
@@ -180,9 +183,33 @@
     "Did you hear that? It said --\n\nCosmic background radiation is a riot!",
     "Well done solving those puzzles.\nPray return to the Waking Sands.",
     "Having trouble finding your checks?\nTry the PopTracker pack!\nIt's got auto-tracking and a detailed map.",
-
-    "Hints suggested by:\nIHNN, Beaker, MrPokemon11, Ember, TheM8, NewSoupVi, Jasper Bird, T1mshady, "
-    "KF, Yoshi348, Berserker, BowlinJim, oddGarrett, Pink Switch, Rever, Ishigh, snolid.",
+    "If you can't see the jungle for the trees, perhaps you should leaf.",
+    "Every puzzle has exactly 5 different solutions.\nThat doesn't seem right? That's because it's not!",
+    "Did you know you can turn off the audio logs by clicking on them again?",
+    "oOwOo *notices your audiolog*",
+    "This radon brought to you by ETAOIN SHRDLU",
+    "Please ignore the previous hint, that was meant for someone else. Networking is so difficult sometimes...",
+    "Oh, nice! I always forget to check this one.",
+    "Progressive Dots can be found. Probably. Unless they can't.",
+    "Don't know why the Mountain King, Mountain King, Mountain King,\nDon't know why the Mountain King has such a stressful halllll...",
+    "Fun Fact: If you laid out every line you've drawn end-to-end, then that would be weird. Why did you do that. That's not how this game works.",
+    "Sorry, this hint was removed by an Eraser.",
+    "I hear the Elevator is a super-handy shortcut if you forgot anything in the Tutorial area. :)",
```

Comment:
```suggestion
    "I hear the Elevator is a super handy shortcut if you forgot anything in the Tutorial area. :)",
```
I don't think a dash here makes sense